tests/test_mappings/test_helpers.py

```
@@ -0,0 +1,39 @@
+from unittest import mock
+
+import pytest
+
+from apps.mappings.helpers import patch_corporate_card_integration_settings
+from apps.workspaces.models import WorkspaceGeneralSettings
+
+
+@pytest.mark.django_db()
+def test_patch_corporate_card_integration_settings(test_connection):
```

Ruff (0.14.3)

Unused function argument: `test_connection`

(ARG001)